esx_service/vmdk_ops.py

```diff
@@ -565,7 +565,7 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     Returns None (if all OK) or error string
     """
     vm_datastore = get_datastore_name(config_path)
-    error_info, tenant_uuid, tenant_name = auth.authorize(vm_uuid, vm_datastore, cmd, opts)
+    error_info, tenant_uuid, tenant_name = auth.get_tenant(vm_uuid)
```

Comment:
Why do we need to call auth.get_tenant here. I think later auth.authorize will call this function inernally?